package.json

```diff
@@ -19,13 +19,14 @@
     "npm": ">=3.0.0"
   },
   "scripts": {
-    "postinstall": "cd lib || builder run npm:postinstall",
```

Comment:
Yes! For `git` installs, we need:

1. `builder` and `builder-victory-component` prod deps
2. a `postinstall` to build `lib` and `dist`

For `npm` installs we:

1. Don't need  `builder` and `builder-victory-component` prod deps
2. The `postinstall` is a noop because `lib` exists, so `cd lib` returns before hitting the `||`

And this scenario above is essentially what `publishr`'s main support goal is. 😉